service/src/test/java/bio/terra/workspace/common/fixtures/ControlledAwsResourceFixtures.java

```diff
@@ -206,47 +209,67 @@ public static ControlledAwsS3StorageFolderResource.Builder makeAwsS3StorageFolde
       (DeleteNotebookInstanceResponse)
           DeleteNotebookInstanceResponse.builder().sdkHttpResponse(SDK_HTTP_RESPONSE_400).build();
 
-  public static final WaiterResponse waiterNotebookResponse =
-      DefaultWaiterResponse.builder()
+  public static final WaiterResponse<DescribeNotebookInstanceResponse> waiterNotebookResponse =
+      DefaultWaiterResponse.<DescribeNotebookInstanceResponse>builder()
           .attemptsExecuted(1)
           .response(DescribeNotebookInstanceResponse.builder().build())
           .build(); // wait successful
-  public static final WaiterResponse waiterNotebookException_1 =
-      DefaultWaiterResponse.builder()
+  public static final WaiterResponse<DescribeNotebookInstanceResponse> waiterNotebookException_1 =
+      DefaultWaiterResponse.<DescribeNotebookInstanceResponse>builder()
           .attemptsExecuted(1)
           .exception(AWS_SERVICE_EXCEPTION_1)
           .build(); // wait failure
-  public static final WaiterResponse waiterNotebookException_2 =
-      DefaultWaiterResponse.builder()
+  public static final WaiterResponse<DescribeNotebookInstanceResponse> waiterNotebookException_2 =
+      DefaultWaiterResponse.<DescribeNotebookInstanceResponse>builder()
           .attemptsExecuted(1)
           .exception(AWS_SERVICE_EXCEPTION_2)
           .build(); // wait failure
 
+  public static String getUniqueInstanceName(String resourceName) {
```

Comment:
If you switch to appendRandNumber for the other PR, we should probably do it here also. It may be too long, but something like "wsm-test-instance-<resource and/or rand-number>" vs "wsm-test-s3-<resource and/or rand-number>" may be helpful

Reply:
Considered the length, but since they are only test resources, did not think it would be a concern. 
Note- adding wsm-test as the prefix for resources so that its easy to identify them on the AWS. 
same approach used in CLI AWS tests 
i.e. cli-test-x-resourceName

Reply:
There is another short-random number function I might as well use that